draft-packages/title-block-zen/KaizenDraft/TitleBlockZen/TitleBlockZen.tsx

```diff
@@ -643,14 +654,28 @@ const TitleBlockZen = ({
         <div className={styles.rowBelowSeparator}>
           <div className={styles.rowBelowSeparatorInner}>
             <div className={styles.rowBelowSeparatorInnerContent}>
-              {(sectionTitle || sectionTitleDescription) &&
-                renderSectionTitle(
-                  sectionTitle,
-                  sectionTitleDescription,
-                  variant,
-                  sectionTitleAutomationId,
-                  sectionTitleDescriptionAutomationId
-                )}
+              {(sectionTitle ||
```

Comment:
One callout I wanted to make on ln 564 that kind of relates to this:
```
          [styles.collapseNavigationArea]:
            collapseNavigationArea &&
            !(sectionTitle || sectionTitleDescription),
```
If we went the render prop route we'd probably want a check there too ^ since you could theoretically have a custom render with out the sectionTitle and sectionTitleDescription prop

Reply:
Great catch 🎉